packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.less

```diff
@@ -0,0 +1,52 @@
+@import (reference) "node_modules/@nova-ui/bits/sdk/less/nui-framework-variables";
+@import (reference) "@nova-ui/bits/sdk/less/nui-framework-variables.less";
+@import (reference) "@nova-ui/bits/sdk/less/mixins.less";
+
+@nui-legend-deemphasized-opacity: 0.3;
+@nui-legend-series-content-margin: 3px;
+
+.legend-description-container {
+    & .legend-description {
+        // line-height: @nui-line-height-default;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
```

Comment:
The mixin `.text-overflow(ellipsis)` could be used to replace these lines. For reference, the mixin is found here: https://github.com/solarwinds/nova/blob/main/packages/bits/src/styles/mixins/text-overflow.less